plugins/woocommerce-admin/client/customer-effort-score-tracks/customer-effort-score-tracks-container.js

```diff
@@ -29,6 +30,12 @@ function CustomerEffortScoreTracksContainer( {
 	resolving,
 	clearQueue,
 } ) {
+	useEffect( () => {
+		if ( queueForPage.length ) {
```

Comment:
`queueForPage` is not defined here yet, so this may always be false anyway.
Should for the `queueForPage` above the `useEffect`.
In that case we will also want to add `queueForPage` to the dependency list of the `useEffect`.